features/step_definitions/supplier_steps.rb

```diff
@@ -100,3 +100,27 @@
   @supplier = get_supplier_with_copyable_service(@framework)
   puts "supplier id: #{@supplier['id']}"
 end
+
+Given /^I have the latest live or standstill framework$/ do
+  response = call_api(:get, "/frameworks")
+  expect(response.code).to eq(200), _error(response, "Failed getting frameworks")
+  frameworks = JSON.parse(response.body)['frameworks']
+  @framework = frameworks.select { |f| f['status'] == 'live' || f['status'] == 'standstill' }.max_by { |f| f['applicationsCloseAtUTC'] }
+  expect(frameworks).not_to be_empty, _error(response, "No live frameworks found")
```

Comment:
So in our [framework application scenario step](https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/step_definitions/supplier_steps.rb#L8), we have a `skip_this_scenario` method that is called if there's no `coming` or `open` frameworks. We should do the same thing here, so the scenario only gets run if the DB is in `standstill` or `live` (and more importantly, doesn't fail otherwise!).

Reply:
implemented in 2122b0d